test/conformance/api/v1alpha1/service_test.go

```diff
@@ -295,7 +295,7 @@ func TestReleaseService(t *testing.T) {
 	const (
 		expectedFirstRev  = test.PizzaPlanetText1
 		expectedSecondRev = test.PizzaPlanetText2
-		expectedThirdRev  = test.HelloWorldText
```

Comment:
The intent here is for the third revision to have unique output that is different from the first two so we can distinguish output is being directed to the correct revision.

For this test we could probably just update ENV vars instead of the image to get the same result.

Reply:
Reused the SingleThreaded image here.